specification/billing/resource-manager/Microsoft.Billing/stable/2024-04-01/types.json

```diff
@@ -0,0 +1,428 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BillingManagementClient",
+    "description": "Documentation for Microsoft.Billing.",
+    "contact": {
+      "name": "Microsoft.Billing",
+      "url": "https://learn.microsoft.com/en-us/rest/api/billing/",
+      "email": "pacebpexphot@microsoft.com"
+    },
+    "version": "2024-04-01"
+  },
+  "paths": {},
+  "definitions": {
+    "Error": {
+      "description": "The error detail.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "The error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "The error message.",
+          "type": "string",
+          "readOnly": true
+        },
+        "target": {
+          "description": "The error target.",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "The error details.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        },
+        "additionalInfo": {
+          "description": "The resource management error additional info.",
+          "type": "array",
+          "items": {
+            "description": "The resource management error additional info.",
+            "type": "object",
+            "properties": {
+              "type": {
+                "description": "The additional info type.",
+                "type": "string",
+                "readOnly": true
+              },
+              "info": {
+                "description": "The additional info.",
+                "type": "object",
+                "readOnly": true
+              }
+            }
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "Resource": {
```

Comment:
In this file there are some types which could be replaced with types defined in [common types](https://github.com/Azure/azure-rest-api-specs/blob/f49258836cf434d10549fe4048fb70f9ea84c4fc/specification/common-types/resource-management/v5/types.json). Any reason you are not using them?

Reply:
Ran a quick scan and I see already wherever needed it is referring common types, I guess your suggestion is to remove any unused references from this file?

Reply:
yes, if it si no longer used it is better to clean the file so that the reviewing process is easier.

Reply:
cleaned up the file and fixed the resource reference